uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Hook/BuildForm/ContactAdvancedSearch.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+class CRM_HRCore_Hook_BuildForm_ContactAdvancedSearch {
+  /**
+   * Checks if the form is the contact advance search and removes
+   * some unnused fields if so.
+   *
+   * @param string $formName
+   * @param CRM_Core_Form $form
+   */
+  public function handle($formName, &$form) {
+    if (!$this->shouldHandle($formName)) {
+      return;
+    }
+
+    $this->removeUnnusedSearchFields($form);
```

Comment:
Seems you wanted to type `removeUnusedSearchFields` instead of `removeUnnusedSearchFields`. There is a typographical error.